src/routes/docs/products/functions/execute/+page.markdoc

```diff
@@ -46,16 +46,15 @@ You can invoke your Appwrite Functions directly from the [Appwrite SDKs](/docs/s
 {% tabsitem #client title="Client SDKs" %}
 
 {% multicode %}
-```js
+```client-web
```

Comment:
very nice catch! can you do the same kind of thing with the others on this page too?